packages/editor/src/client/atoms/descriptionList.tsx

```diff
@@ -44,7 +45,16 @@ export function DescriptionListItem({label, children}: DescriptionListItemProps)
         style={{
           marginLeft: 20
         }}>
-        {children}
+        {children || (
```

Comment:
what happens if `children` and `message` are empty? Also you need a way to change the background color if you want to use the same component for error messages.

I suggest to create a new component that reuses the `<DescriptionListItem>`

Reply:
I made an attempt with a new component, look forward to having your review